spk/gentoo-chroot/Makefile

```diff
@@ -0,0 +1,92 @@
+SPK_NAME = gentoo-chroot
+SPK_VERS = 13.0
+SPK_REV = 1
+SPK_ICON = src/gentoo.png
+DSM_UI_DIR = app
+
+DEPENDS = 
+WHEELS = pyextdirect==0.3.1 flask Werkzeug Jinja2 itsdangerous
+SPK_DEPENDS = "python>2.7.3-3"
```

Comment:
I keep getting that version wrong. `"python>=2.7.8-10"` would work, but let's make it what it should actually be: 
SPK_DEPENDS = `"python>=2.7.6-8"`